index.html

```diff
@@ -50,7 +50,7 @@ <h5 class="card-title">Roadmap</h5>
         <h5 class="card-title">Documentation</h5>
         <ul>
           <li>
-            <a href="https://www.opendevstack.org/ods-documentation/common/latest/local-install/overview.html">Getting started - installation</a>
+            <a href="https://www.opendevstack.org/ods-documentation/common/latest/administrator-guides/install-requirements.html">Getting started - installation</a>
           </li>
           <li>
             <a href="https://www.opendevstack.org/ods-documentation/common/latest/development-guide.html">ODS Development Guide</a>
```

Comment:
Can you please change the link _ODS Development Guide_  to https://www.opendevstack.org/ods-documentation/common/latest/contributor-guides/development.html